test/utils.h

```diff
@@ -483,6 +483,72 @@ class NVFuserTest : public ::testing::Test {
   bool capturing_ = false;
 };
 
+// Fixture with param class must be uniquely identified, i.e., can't be in an
+// anonymous namespace
+template <typename tParam>
+class NVFuserFixtureParamTest : public ::testing::TestWithParam<tParam> {
```

Comment:
Instead of copy-pasting the code from `NVFuserTest`, should we just use multiple inheritance:
```C++
template <typename tParam>
class NVFuserFixtureParamTest : public WithParamInterface<tParam>, public NVFuserTest {
```

Reply:
Also, should we change this
https://github.com/NVIDIA/Fuser/blob/52b98c77cfe312ab7a60171f4befd292c42c234f/test/test_memory.cpp#L29-L31
to use `NVFuserFixtureParamTest`?

Reply:
You are right, I focused too much on rewriting the tests that I overlooked this stop-gap solution.

I will update the code in the `test/utils.h` and in `test/test_memory.cpp`.

Thanks!